packages/devtools_app/lib/src/screens/performance/panes/timeline_events/legacy/legacy_event_processor.dart

```diff
@@ -0,0 +1,233 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
the code in this file was moved from timeline_event_processor.dart (reviewers - you may want to review there first to see what you're looking for in this file).